app/javascript/packs/pf_form.scss

```diff
@@ -1,5 +1,6 @@
 @import '~@patternfly/patternfly/components/Button/button.css';
 @import '~@patternfly/patternfly/components/Check/check.css';
+@import '~@patternfly/patternfly/components/Dropdown/dropdown.css';
```

Comment:
This will include dropdown.css into all existing forms, but Dropdown is not even a form element (contrary to FormSelect). I'd suggest creating a new pack `cms.scss` that will be loaded automatically here:
https://github.com/3scale/porta/blob/3eaf1005b910b55c04f61bf411240a8621ec465c/app/views/layouts/_cms.html.erb#L4

This has a very low impact in performance, but still.